Formula/c/cryptography.rb

```diff
@@ -52,17 +48,22 @@ def pythons
 
   def install
     ENV.append_path "PATH", buildpath/"bin"
+    # Resources need to be installed in a particular order, so we can't use `resources.each`.
+    resources_in_install_order = %w[setuptools setuptools-rust semantic-version]
```

Comment:
Does it still need `setuptools-rust` and `semantic-version`? I thought `maturin` replaced those.